File: Heat/ViewModels/Plants/DetailsIdentifyPlantViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Plants

    Public Class DetailsIdentifyPlantViewModel

        Property ID As Integer

        <Display(name:="Nominativo")> _
        Property Name As String

        <Display(name:="Codice dell'impianto per la provincia")> _
        Property PlantDistinctCode As String

        <Display(name:="Indirizzo")> _
        Property Address As String

        <Display(name:="Numero Civico")> _
        Property StreetNumber As String

        <Display(name:="Palazzo")> _
        Property Building As String

        <Display(name:="Scala")> _
        Property Stair As String

        <Display(name:="Interno")> _
        Property Apartment As String

        <Display(name:="Località")> _
        Property City As String

        <Display(name:="CAP")> _
        Property PostalCode As String

        <Display(name:="Area")> _
        Property Area As String

        <Display(name:="Zona")> _
        Property Zone As String

        <Display(name:="Provincia")> _
        Property District As String

    End Class

End Namespace

File: Heat.ConvertedToC#/ViewModels/Plants/DetailsIdentifyPlantViewModel.cs
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Plants
{

    public class DetailsIdentifyPlantViewModel
	{

		public int ID { get; set; }

		[Display(Name = "Nominativo")]
		public string Name { get; set; }

		[Display(Name = "Codice dell'impianto per la provincia")]
		public string PlantDistinctCode { get; set; }

		[Display(Name = "Indirizzo")]
		public string Address { get; set; }

		[Display(Name = "Numero Civico")]
		public string StreetNumber { get; set; }

		[Display(Name = "Palazzo")]
		public string Building { get; set; }

		[Display(Name = "Scala")]
		public string Stair { get; set; }

		[Display(Name = "Interno")]
		public string Apartment { get; set; }

		[Display(Name = "Località")]
		public string City { get; set; }

		[Display(Name = "CAP")]
		public string PostalCode { get; set; }

		[Display(Name = "Area")]
		public string Area { get; set; }

		[Display(Name = "Zona")]
		public string Zone { get; set; }

		[Display(Name = "Provincia")]
		public string District { get; set; }

	}

}
